client/dist/style.css

```diff
@@ -132,6 +152,124 @@
   bottom: 15px;
 }
 
+/* Product Info Styles  =====================================================*/
+.styles-thumb {
+  height: auto;
+  width: 55px;
+  overflow: hidden;
+  margin: 0 auto;
+}
+
+.style-title {
+  font-size: larger;
+  margin-top: 20px;
+  margin-bottom: 10px;
+}
+
+.style-title, .product-info-align {
+  margin-left: 8px;
+}
+
+.product-top {
+  margin-top: 50px;
+}
+
+.img-cont {
+  height: 55px;
+  width: 55px;
+  border-radius: 50%;
+  overflow: hidden;
+  background-color: black;
+  margin: 8px;
+}
+
+.img-cont.active {
+  box-shadow: 5px 5px 5px cyan;
+}
+
+.red {
```

Comment:
.red and .badge are kind of vague class names. Maybe change them to something a little more descriptive. Also they could cause conflicts with others components

Reply:
I have changed .red =>  .sale-price and badge => .style-tnail